scripts/heroku_sync_db_to_local.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/zsh
+# Script for syncing remote DB to local
+
+APP_NAME=$1
+DB_NAME=$2
+DUMP_FILE='latest_backup.dump'
+
+if [ -n "$APP_NAME" ] && [ -n "$DB_NAME" ]; then
+
+  # Create new backup
+  echo "Creating backup for $APP_NAME..."
+  heroku pg:backups capture --app $APP_NAME
```

Comment:
I have added this and the directory name default to the 'to do' section in the read me.